packages/functions/src/donor/BookAppointmentHandler.test.ts

```diff
@@ -135,6 +136,13 @@ test("Valid request books appointment", async () => {
 
   expect(data.bookedAppointment.id).toEqual(APPOINTMENT_TO_BOOK_2);
   expect(data.bookedAppointment.donorId).toEqual(DONOR_ID);
+
+  expect(appointment[0].lastChangeType).toEqual(BookingChange.BOOKED);
+  expect(
+    Date.now() - (appointment[0]?.lastChangeTime?.toMillis() || 0)
```

Comment:
Why do you need the `|| 0`? 

Can it ever mask a real problem?

Reply:
The CI/CD cries the variable might be undefined.
It won't ever mask a real problem, since undefined will return 
(UNIX - 0 < 3_000 == false)

and will crash, as expected from an undefined lastChangeTime.